### DISCUSSION TITLE
Unique computed and personal data for every tour

### DISCUSSION THREAD

**Discussion by Atombike** [mytourbook/mytourbook#1408] [Ideas]
Hi,
Could you make the following data unique for every tour?
I'm using Mytourbook for several years and a lot of personal data changed since then.
However there is also differences between spring and autumn. Eg I can lose 10kg from my weight till the end of season.
Other problem I found is altitude calculation. It would be great if I can modify the formula for only 1 tour. If I make changes to fix some issue on some tour it will mess up older tour which was ok.

- Weight - this can be updated manually on every tour but would be great to modify monthly
- HR zones, max HR
- altitude calculation formula
- if not a problem, pleas add more field where I can define more equipment. EG shoe types. Currently I'm using tour type for this and tags for the recording device.

**Comment by wolfgang-ch** [comment]
Can you create feature requests for every item separately and provide more details and screenshots
